src/config/config.ts

```diff
@@ -826,7 +826,7 @@ const defaultFlagToCached = async (flag: CompletableOptionFlag<any>, isWritingMa
   // if not specified, try the default function
   if (typeof flag.default === 'function') {
     try {
-      return await flag.default({options: {}, flags: {}}, isWritingManifest)
+      return await flag.default({options: flag, flags: {}}, isWritingManifest)
```

Comment:
Need to pass the flag definition to default function to allow the function to access options from the flag